client/src/main/kotlin/app/cash/backfila/client/Connectors.kt

```diff
@@ -5,6 +5,10 @@ object Connectors {
   const val ENVOY = "ENVOY"
 }
 
+data class PerRunOverrideData(
+  val overrideClusterType: String? = null,
+)
+
 data class HttpConnectorData(val url: String)
 
-data class EnvoyConnectorData(val clusterType: String)
+data class EnvoyConnectorData(val clusterType: String, val gnsLabel: String? = null)
```

Comment:
If specified, allow for setting the GNS label header for envoy clients per https://docs.google.com/document/d/1zKK12F1tesOcDDm8GnYCZbQAI-MlVwtyRxDYr8SJ_5w/edit#heading=h.xinrsz8k2wnd